components/automate-chef-io/content/docs/applications-setup.md

```diff
@@ -106,8 +103,8 @@ Chef Automate's TLS certificate file is located at `/hab/svc/automate-load-balan
 
 From Chef Automate:
 
-1. On the command line, run `chef-automate config show`
-1. Copy the `[[global.1.frontend_tls]]` certificate contents from the certificate list
+1. On the command line, run `chef-automate external-cert show`
```

Comment:
Is this change in the dev channel?

Reply:
Yes, it's in dev and also in acceptance. It comes from https://github.com/chef/automate/pull/2504

Reply:
Thanks!